Gemfile.lock

```diff
@@ -245,13 +245,13 @@ GEM
     nap (1.1.0)
     naturally (2.2.1)
     netrc (0.11.0)
-    nokogiri (1.12.3)
+    nokogiri (1.12.4)
       mini_portile2 (~> 2.6.1)
       racc (~> 1.4)
     octokit (4.21.0)
       faraday (>= 0.9)
       sawyer (~> 0.8.0, >= 0.5.3)
-    oj (3.13.1)
+    oj (3.13.4)
```

Comment:
These gems patch version changes are a byproduct of my dance with the release toolkit version. They snuck in there because I had to go through two versions, so instead of `git revert` two commits I manually rolled back the `Pluginfile` and run `bundle update fastlane-plugin-wpmreleasetoolkit`. 

I don't think they present any issue, so I decided not to spend any extra time to amend them.